lib/classes/Service.js

```diff
@@ -46,17 +46,18 @@ class Service {
       return BbPromise.resolve();
     }
 
-    return BbPromise.all([
-      serverlessConfigFileUtils.getServerlessConfigFilePath(this.serverless),
-      serverlessConfigFileUtils.getServerlessConfigFile(this.serverless),
-    ])
-      .then(args => this.loadServiceFileParam(...args))
-      .catch(error => {
-        if (this.serverless.cli.isHelpRequest(this.serverless.processedInput)) {
-          return null;
-        }
-        throw error;
-      });
+    try {
+      const args = await BbPromise.all([
```

Comment:
Either we should completely refactor to native promises or leave function body as it is, this is partial convert that leaves us somewhere mid-step.

Task was simply about marking _async_ function as _async functions_, but leave their body without any changes implied.

So let's maybe revert all in body changes

Reply:
Just to clear and start working again, all we need to do is to identify async functions and mark them as `async function ()` sorry if I am misunderstanding or causing inconvenience

Reply:
@Edu93Jer Yes, I believe it's quite clearly explained in main issue description. If you feel some of the explanations are vague or raise questions, please comment over there. We need to be sure there are no doubts in understanding the goal of the issue